tests/client/amo/helpers.js

```diff
@@ -50,8 +50,9 @@ export const fakeReview = {
 /*
  * Redux store state for when a user has signed in.
  */
-export const signedInApiState = {
+export const signedInApiState = (overrides = {}) => ({
```

Comment:
Nothing mutated it but it didn't very safe to be sharing that object across tests in multiple files. `Object.freeze` should work too. I'll do that.